lua/cmp/config/default.lua

```diff
@@ -41,6 +41,7 @@ return function()
       completeopt = 'menu,menuone,noselect',
       keyword_pattern = [[\%(-\?\d\+\%(\.\d\+\)\?\|\h\w*\%(-\w*\)*\)]],
       keyword_length = 1,
+      docs_initially_visible = true,
```

Comment:
Hm... I'll consider the better place for this option.

Reply:
Yeah I was unsure about it too. IMO it doesn't make sense to put it in `WindowConfig` since those only involve UI settings, and all other categories don't feel right either...

Reply:
We can also just get rid of this option and let users configure if docs should be initially visible or not.